src/css/custom.scss

```diff
@@ -9,7 +9,7 @@
 @import url('https://fonts.googleapis.com/css2?family=IBM+Plex+Mono&family=IBM+Plex+Sans:wght@400;700&family=Press+Start+2P&display=swap|');
 
 :root {
-    --ifm-color-primary: #ff0077;
+    --ifm-color-primary: yellow;
```

Comment:
While this does pop more, there needs to be a discussion around the wiki stack/design/styling/look & feel, so this ought to revert to the original color.

Reply:
Yeah, I agree. Particularly after looking at it a little more the yellow is pretty harsh on the eyes and too much contrast on the subtitles. We should stick to the purple.